hazelcast-client/src/test/java/com/hazelcast/client/usercodedeployment/ClientUserCodeDeploymentTest.java

```diff
@@ -161,10 +161,13 @@ private void assertCodeDeploymentWorking(HazelcastInstance client, EntryProcesso
         for (int i = 0; i < keyCount; i++) {
             map.put(i, 0);
         }
-        map.executeOnEntries(entryProcessor);
+
+        for (int i = 0; i < 5; i++) {
```

Comment:
can you put an explanation why this number 5 for the test so that test maintainer can understand better the significance of this number?